sonar-db/src/main/java/org/sonar/db/user/GroupDao.java

```diff
@@ -110,4 +119,17 @@ private static String groupSearchToSql(@Nullable String query) {
   private GroupMapper mapper(DbSession session) {
     return session.getMapper(GroupMapper.class);
   }
+
+  private static class SelectByNames implements Function<List<String>, List<GroupDto>> {
+    private final GroupMapper mapper;
+
+    private SelectByNames(GroupMapper mapper) {
+      this.mapper = mapper;
+    }
+
+    @Override
+    public List<GroupDto> apply(@Nonnull List<String> partitionOfNames) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this "Nonnull" annotation to honor the overridden method's contract. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS2638)